styles/message.mcss

```diff
@@ -201,26 +204,20 @@ Message {
   }
 
   footer {
-    margin: 10px 10px;
-    border-top: 1px solid #eee;
-    padding-top: 10px;
-    margin-top: 5px;
-    padding-left: 5px;
+    margin: 5px 0 20px;
+    padding: 0 20px
```

Comment:
So looking at this again, and for longer, I can see why the padding is the way it is. Makes sense when there are multiple messages and you want to distinguish which message the actions are for.

Still not quite sure what this means for the case above though (where there's only one message).